install_config/web_console_customization.adoc

```diff
@@ -855,6 +855,32 @@ xref:../install_config/configuring_authentication.adoc#OpenID[OpenID] identity
 providers, which require visiting an external URL to destroy single sign-on
 sessions.
 
+[[changing-the-membership-whitelist]]
```

Comment:
We should move this section up higher in the topic to the where the other extensions are described.